.github/workflows/codegen-test.yml

```diff
@@ -9,9 +9,12 @@ env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
-  downstream-aws:
-    name: Test AWS Downstream
+  downstream-test:
```

Comment:
This is refactored to just require new providers to be added to the list rather than entire codeblocks to be added